oma-refresh/src/verify.rs

```diff
@@ -114,12 +115,19 @@ pub fn verify<P: AsRef<Path>>(
         }
     }
 
-    let p = StandardPolicy::new();
-    let mut v = VerifierBuilder::from_bytes(s.as_bytes())?.with_policy(
-        &p,
-        None,
-        InReleaseVerifier::new(&cert_files, mirror)?,
-    )?;
+    // Derive p to allow configuring sequoia_openpgp's StandardPolicy.
+    let mut p: StandardPolicy<'_> = StandardPolicy::new();
```

Comment:
不建议在类型可自动推导的情况下指定带生命周期的类型

```suggestion
    let mut p = StandardPolicy::new();
```